### DISCUSSION TITLE
It seems to work but

### DISCUSSION THREAD

**Discussion by kbthomas59** [cariboulabs/cariboulite#93] [Q&A]
I have the device apparently working, but I never see a signal in waterfall plots and never hear anything but static. I have tried various antennas on both 6GHz connector (HiF) and S1G.  This has to be something simple. It set up easily on CubicSDR and gnuradio-companion, both via SoapySDRServer.  I don't see errors in the terminal where SoapySDRServer is running or in the terminal where the receiver software is running.  I do see that RTL-SDR works with the same antennas.

**Comment by edegraaff** [comment]
Sounds similar to my issue

**Reply by K7MDL2** [reply]
I get the same symptoms with SDR++ running SoapySDR.  If you run sdrpp --server and run sdrpp as a client (local or on a remote box) with sdr++ Server selected, you can connect and you get full audio and spectrum.  Other apps I have tried local and remote have been a mixed bag of nothing, stuttered audio/visual, or fully working.
